flare/qe_util.py

```diff
@@ -9,7 +9,7 @@
 def run_espresso(qe_input, structure, pw_loc):
     run_qe_path = qe_input
     edit_qe_input_positions(run_qe_path, structure)
-    qe_command = 'mpirun {0} < {1} > {2}'.format(pw_loc, run_qe_path,
+    qe_command = '{0} < {1} > {2}'.format(pw_loc, run_qe_path,
```

Comment:
This is a good catch, puts more power in the hand of the user to define the PWSCF command with/without MPI